packages/editor/src/components/media-categories/index.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { resolveSelect } from '@wordpress/data';
+import { decodeEntities } from '@wordpress/html-entities';
+
+/**
+ * Internal dependencies
+ */
+import { store as coreStore } from '@wordpress/core-data';
+
+const getExternalLink = ( url, text ) =>
+	`<a href="${ url }" target="_blank" rel="noreferrer noopener">${ text }</a>`;
+
+const getOpenvereLicense = (
+	license,
+	licenseVersion = '', // unknown version.
+	licenseUrl
+) => {
+	let licenseName = license.trim();
+	// PDM has no abbreviation
+	if ( license !== 'pdm' ) {
+		licenseName = license.toUpperCase().replace( 'SAMPLING', 'Sampling' );
```

Comment:
Do I need this for GB? If yes, what's that about and what are the translation needs?

Reply:
I'm not sure I fully understand the question. This string replacement is just getting rid of some all-caps that appears in some old versions of CC licenses.